app/v1/models/request/NinoApplication.scala

```diff
@@ -62,7 +62,7 @@ object NinoApplication {
   private val officeNumberPath = __ \ "officeNumber"
   private val contactNumberPath = __ \ "contactNumber"
   private val countryPath = __ \ "country"
-  private val namePath = __ \ "name"
+  private val namesPath = __ \ "name"
```

Comment:
```suggestion
  private val namesPath = __ \ "names"
```
sorry if I wasn't clear in my first comment

Reply:
Ignore this. This will be done under another story